backend/validators/models.py

```
@@ -135,6 +163,39 @@ def __str__(self):
+    )
+    observed_at = models.DateTimeField(db_index=True)
+    onchain_status = models.CharField(max_length=20, choices=ValidatorWallet.STATUS_CHOICES)
+    metrics_status = models.CharField(max_length=10, choices=ValidatorWallet.GRAFANA_STATUS_CHOICES)
+    logs_status = models.CharField(max_length=10, choices=ValidatorWallet.GRAFANA_STATUS_CHOICES)
+    version_status = models.CharField(
+        max_length=10, choices=ValidatorWalletStatusSnapshot.VERSION_STATUS_CHOICES, default='unknown'
+    )
+    node_version = models.CharField(max_length=50, blank=True)
+
+    class Meta:
+        ordering = ['-observed_at']
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)